README.md

```diff
@@ -1,2 +1,379 @@
 # pks-micro-integrator
+
 This repository contains Micro Integrator Pivotal PKS artifact resources.
+
+
+# Prerequisites
+
+* An already setup [Pivotal Application service(PAS) (v2.7.1)](https://network.pivotal.io/products/elastic-runtime#/releases/477769)
+* An already setup [Pivotal Container Service(PKS) (v1.5.1)](https://network.pivotal.io/products/pivotal-container-service#/releases/481414)
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/) version [nginx-0.22.0](https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.22.0) on the PKS provisioned Kubernetes cluster
```

Comment:
@ThilinaManamgoda do we still have this restriction? Based on @Aaquiff's findings (see https://github.com/wso2/kubernetes-ei/issues/216) can't we eliminate this version restriction?

Reply:
We only test this with version 0.22.0. Until we test this with later versions I think we have to go with this one.

Reply:
@ThilinaManamgoda I think it is better if we can create an issue and track this change.

Reply:
sure will do